### DISCUSSION TITLE
[#1012824] Editor mode breaking OreUI

### DISCUSSION THREAD

**Discussion by xKingDark** [Mojang/minecraft-editor#105] [Bugs]
**How to Reproduce**:
  **1.** Select an area using the **Selection** tool
  **2.** Go into Crosshair mode and run **/reload**
  **3.** Run /reload again so Editor Mode starts the default extensions
  **4.** Switch to the Test Extension
  
**Observed Result**:
<img src="https://user-images.githubusercontent.com/47607185/233102259-1ec3351a-db82-4fde-ad75-07110605726a.png">

Going back to the start screen appears like this for a few second
<img src="https://user-images.githubusercontent.com/47607185/233102416-52f747d3-8bdf-45a3-8c38-848595b161ba.png">

**Expected Result**:
Nothing should break.

**Helpful Info**:
- Preview build version: `1.20.0.20`

Extension used: [Editor_Extension.zip](https://github.com/Mojang/minecraft-editor/files/11274010/Editor_Extension.zip)

**Comment by minecraftchloe** [comment]
Should be fixed! Let us know if this is an issue again!
